src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityEvaporationPool.java

```diff
@@ -0,0 +1,104 @@
+    }
+
+    protected BlockPattern createStructurePattern() {
+        return FactoryBlockPattern.start()
+                .aisle("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "                                ")
+                .aisle("ABBBBBBBBBBBBBBBBBBBBBBBBBBBBBBA", " BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
+                .aisle("ABCCCCCCCCCCCCCCCCCCCCCCCCCCCCBA", " B                            B ")
```

Comment:
You can replace all of these lines with a single one of them, and then call `setRepeatable(int)` to make this more concise.